perllib/FixMyStreet/App/Controller/Admin/ExorDefects.pm

```diff
@@ -140,6 +140,18 @@ sub download : Path('download') : Args(0) {
 
         foreach my $report (@$inspections) {
             my ($eastings, $northings) = $report->local_coords;
+
+            my $location = "${eastings}E ${northings}N";
+            $location = "[DID NOT USE MAP] $location" unless $report->used_map;
+            my $closest_address = $c->cobrand->find_closest($report);
+            if ($closest_address) {
+                # Match code in bin/oxfordshire/open311_service_request.cgi
```

Comment:
As we're doing this twice, could we not pass a flag to the `find_closest` function to just return the address minus the other cruft that we're stripping out here?